### ISSUE TITLE
Steam doesn't count the play time when playing the vr mod

### ISSUE THREAD

**Issue by stjema** [KozGit/DOOM-3-BFG-VR#343]
What are the steps to reproduce this?

**Comment by sycspysycspy** [comment]
it is a different exe, it doesnt even require steam to run. so yeah no play time recorded, i dont think it is an issue, it is a feature.

**Comment by stjema** [comment]
would you tell me howto replicate this with any steam game?

**Comment by sycspysycspy** [comment]
I don't think it is possible unless you mod the exe files for other games like this mod does. or have you tried to run steam offline? I think the time recored during the offline period will not be acknowledged by steam until you go online.
